src/test/java/emissary/util/DisposeHelperTest.java

```diff
@@ -22,6 +23,9 @@
 
 class DisposeHelperTest {
 
+    static {
```

Comment:
This solution works, but we should avoid using a static class initialization block and instead use the common JUnit pattern of performing "before all tests" setup in a static method that's annotated with the `org.junit.jupiter.api.BeforeAll` annotation.  _(We should also move this new method to be just above the existing `setup()` method.)_

```suggestion
    @BeforeAll
    static void setupClass() {
```

Reply:
Got it.